tests/ui/union/union-derive-clone.stderr

```diff
@@ -29,6 +29,9 @@ note: trait bound `CloneNoCopy: Copy` was not satisfied
    |
 LL | #[derive(Clone, Copy)]
    |          ^^^^^ unsatisfied trait bound introduced in this `derive` macro
+LL | union U5<T> {
+   |          - implicit unsatisfied bound on this type parameter
+   = note: `#[derive(Clone)]` introduces `where`-bounds requiring every type parameter to implement `Clone`
```

Comment:
This error message looks misleading. The `T: Clone` bound of `#[derive(Clone)]` is required for `Clone` and the problem is the `U5<CloneNoCopy>: Copy`, not `U5<CloneNoCopy>: Clone`